dist/api.classic.js

```diff
@@ -250,8 +250,12 @@
             adapters[adapterName] = definition;
             adapter = definition;
         },
-        startScan: function(serviceUUIDs, foundFn, completeFn, errorFn) {
+        startScan: function(serviceUUIDs, foundFn, completeFn, errorFn, options) {
```

Comment:
IMO an options object should be used to replace all args. Can you stick with arguments for this addition and just have an allowDuplicates argument?